Cargo.toml

```diff
@@ -10,7 +10,7 @@ homepage = "https://github.com/dusk-network/jubjub"
 license = "MIT/Apache-2.0"
 name = "dusk-jubjub"
 repository = "https://github.com/dusk-network/jubjub"
-version = "0.8.1"
+version = "0.8.2"
```

Comment:
Please don't upgrade the version yet here.

Reply:
Agreed, this is has to be done during the release process.